app/create/page.tsx

```diff
@@ -172,6 +197,11 @@ const Page = () => {
             `ipfs://${metadata_upload_resData.IpfsHash}/`,
           ])
         );
+        // also call the set LOCKTIME FUNCTION HERE
+      const set_campaign_locktime =   await tokenbound.lock({
+        tbaAddress: campaignAddress,
+        lockUntill: dateTimeToUnixTimestamp
+     });
 
```

Comment:
Since the lock function is optional, the `setLoadingPercentage` should to to 100 if the user is locking the campaign and if not.

Reply:
It `setLoadingPercentage` still does not get to 100 if i am not locking the campaign